lib/evm_logic.rb

```diff
@@ -348,7 +357,7 @@ def calculate_actual_cost(costs)
     # @param [hash] hash_with_data target issues of EVM
     # @return [array] EVM hash. Key:time, Value:EVM value
     def convert_to_chart(hash_with_data)
-      hash_converted = Hash[hash_with_data.map { |k, v| [k.to_time.to_i * 1000, v] }]
+      hash_converted = Hash[hash_with_data.map { |k, v| [k.to_time(:local).to_i * 1000, v] }]
```

Comment:
__[rubocop]__
* Line is too long. [93/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)